x-pack/plugins/alerting/server/routes/rule/apis/bulk_delete/bulk_delete_rules_route.ts

```diff
@@ -22,10 +22,7 @@ export const bulkDeleteRulesRoute = ({
     {
       path: `${INTERNAL_BASE_ALERTING_API_PATH}/rules/_bulk_delete`,
       validate: {
-        body: schema.object({
-          filter: schema.maybe(schema.string()),
-          ids: schema.maybe(schema.arrayOf(schema.string(), { minSize: 1, maxSize: 1000 })),
-        }),
+        body: bulkDeleteRulesRequestParamsSchemaV1,
```

Comment:
we need to assert the return type of this route as well (line 36)

Reply:
Fixed: a5f18a611de7a7736836361602d7258b299f92be

Reply:
nitpick: this should be `bulkDeleteRulesRequestBodySchemaV1`, since its the body for the request, not params